client/src/pages/Login/Login.tsx

```diff
@@ -69,23 +75,21 @@ const Login: React.FC = () => {
           <Title>Login</Title>
           <InputGroup>
             <Label>Email</Label>
-            <Input type="email" {...register("email")}/>
+            <Input type="email" {...register("email")} />
             <LoginErrorContainer>
               {errors.email && errors.email?.message && (
-              <p>{errors.email.message}</p>
+                <p>{errors.email.message}</p>
               )}
             </LoginErrorContainer>
-            
           </InputGroup>
           <InputGroup>
             <Label>Password</Label>
-            <Input type="password" {...register("password")}/>
+            <Input type="password" {...register("password")} />
             <LoginErrorContainer>
               {errors.password && errors.password?.message && (
-              <p>{errors.password.message}</p>
+                <p>{errors.password.message}</p>
```

Comment:
Not sure why any of these changes were made lol I think my vscode formatted it a bit different and it got mixed in with the commit - You can ignore changes in this file other than the addition of useNavigate hook